src/components/EventCard/EventCard.js

```diff
@@ -1 +1,18 @@
-import React from 'react';
\ No newline at end of file
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const EventCard = ({ id, name, image, city, state, date }) => {
+  return(
+    <Link to={`/event/${id}`}>
+      <div className='event-card' style={{ backgroundImage: `url(${image})`}}>
+        <div className='text-container'>
+          <h3 className='card-name'>{name}</h3>
+          <h5 className='card-location'>{city}, {state}</h5>
+          <h5 className='card-date'>{date}</h5>
+        </div>
+      </div>
+    </Link>
+  )
+}
+
+export default EventCard;
```

Comment:
Nice use of inline styles here for the backgroundImage! I haven't used that before. Also glad we decided to just make each card a Link. Looks great.